pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -117,16 +124,21 @@ type BundleTarget struct {
 }
 
 // AdditionalFormats specifies any additional formats to write to the target
+// +structType=atomic
```

Comment:
@erikgb same here, does not have to be atomic, right?

Reply:
Fixed